### DISCUSSION TITLE
Savings Quantization Presets v1.8

### DISCUSSION THREAD

**Discussion by themonkeybars** [djphazer/O_C-Phazerville#99] [Q&A]
Running in Hemispheres mode, using DualTuring on both sides, is there a way to save the quantization settings , every time i set it up and save the preset, then power cycle, the quantization settings are lost?

Running the following build - (o_C-phazerville-v1.8_phz-ed61862c.hex)

**Comment by djphazer** [comment]
Currently, you have to save [quantizer settings](https://firmware.phazerville.com/Hemisphere-Quantizer-Setup) as a **Calibr8or** preset and they will be reloaded (the first 4 anyway). I think I can hack it to make this happen automatically... soon.

**Reply by cardinal-bin** [reply]
I don't quite understand how this works in 1.81, should it save the quantizer preset when you save a hemisphere preset?

Also, just a thought, but could the second set of 4 quantizers be taken from the second Calibr8 preset..? That way you could save all 8.

**Reply by djphazer** [reply]
@cardinal-bin yep, so in v1.8.1, saving a Hemisphere Preset automatically calls the save function in Calibr8or. It's a bit of a hack, but it works for now in most cases... imagine a patch cable that just barely reaches from one table to another in the same room.

Trying to save the other 4 Q-engine settings into a different preset sounds like a whole can of worms that's more trouble than it's worth... imagine trying to run a patch cable into a different room entirely! I know it's limited, but for now, you can customize the 4 USER-defined scales which occupy Q5-Q8 by default.

I was originally only going to include 4 global quantizer engines because of the settings storage limitations, so consider the other ones a bonus. I might eventually restructure the EEPROM data to include all 8 quantizer settings in the GlobalSettings. Newer 8-channel O_C hardware also accommodates more settings.
